docs/report/report.tex

```diff
@@ -188,9 +188,80 @@ \section{Design Mockups}
 
 \section{Durchführung}
 
-\subsection{Technologien und Hilfsmittel}
+\subsection{Technologien}
+Zu Beginn des Projekts diskutierte das Team die Technologien, mit denen die Smart Music Box umgesetzt werden sollte.
+Der Kunde ließ dem Projektteam für die Umsetzung der Smart Music Box sehr viele Freiheiten.
+Es wurde lediglich die Anforderung gestellt, dass die Applikation neben PCs und Laptops auch für mobile Endgeräte optimiert sein soll.
+Auf Basis dieser Anforderung evaluierte das Projektteam bekannte Technologien und recherchierte mögliche Alternativen.
+\colorbox{yellow}{Wo Projektstruktur --> Monorepo?}
+
+
+\subsubsection{Programmiersprachen}
+
+\subsubsection*{TypeScript}
+Da alle Teammitglieder bereits Erfahrungen mit JavaScript gesammelt hatten wurde diese zuerst als Programmiersprache vorgeschlagen.
+Aufgrund von nicht vorhandener Typisierung wurde dann allerdings die von Microsoft entwickelte Sprache TypeScript für die Projektierung genutzt. Da diese aufgrund des
+Aufbaus auf ECMAScript-6-Standards eine große syntaktische Ähnlichkeit zu JavaScript besitzt, fiel das Erlernen den Entwicklern relativ leicht.
+Bei der Kompilierung von TypeScript-Code wird dieser zu gültigem JavaScript Code, wodurch auf eine Vielzahl von bereits existierenden JavaScript-Paketen zugegriffen werden konnte.
+Außerdem lässt sich Typescript durch Nutzung der Laufzeitumgebung Node.js auch als Backendsprache verwenden und ersparten dem Team so die Nutzung von verschiedenen Programmiersprachen.
+
+\subsubsection{Frameworks}
+
+\subsubsection*{Vue.js}
+Zur Gestaltung der Verwaltungsoberfläche für die Music-Box, die zum Verwalten der NFC-Tags und steuern der Ausgabegeräte genutzt werden sollte einigte sich das Team außerdem auf die Nutzung des
+clientseitigen JavaScript-Webframeworks \textit{Vue.js}\footnote{\raggedright\url{https://vuejs.org/}}
+Dies wurde gewählt, da es sehr gut dokumentiert ist und eine flache Lernkurve besitzt, was es gerade für die Gruppenmitglieder mit weniger Erfahrung im Web-Bereich leichter zu erlernen machte.
```

Comment:
```suggestion
Dies wurde gewählt, da es sehr gut dokumentiert ist und eine flache Lernkurve besitzt, wodurch der Einstieg für Teammitglieder mit weniger Web Erfahrung leichter war.
```

Reply:
Habe es geändert zu: 

"wodurch der Einstieg für Teammitglieder mit weniger Erfahrung in der Web-Entwicklung leichter war."